README.md

```diff
@@ -20,7 +20,7 @@ The goal of this library is to reimplement GalSim functionalities in pure JAX to
 
 ## Contributing
 
-Everyone can contribute to this project, please refer to the CONTRIBUTING.md document for details. 
+Everyone can contribute to this project, please refer to the [CONTRIBUTING.md](CONTRIBUTING.md) document for details. 
```

Comment:
One quick point.  If you want to eventually use Sphinx for documentation (which I'd recommend), you probably want all of these be .rst rather than .md.  Then Sphinx can render it pretty for you.  We didn't bother I guess for GalSim proper, but it's probably something we should do.

Reply:
Thanks, yeah  I definitely want to use Sphinx to render the doc. I think it's possible to include the changelog even if it's in .md format. They do it for instance for the JAX project: https://github.com/google/jax/blob/main/CHANGELOG.md